e2e/test/scenarios/onboarding/home/browse.cy.spec.js

```diff
@@ -40,4 +40,24 @@ describe("scenarios > browse data", () => {
     cy.findByRole("heading", { name: "Products" }).click();
     cy.findByRole("gridcell", { name: "Rustic Paper Wallet" });
   });
+  it("can see and dismiss the modal explainer banner", () => {
+    cy.visit("/");
+    cy.findByRole("listitem", { name: "Browse data" }).click();
+    const banner = cy
```

Comment:
nit: This could probably just be a selector with a click action, rather than building all the variables. But it's also not that important.